galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.tsq.manager.ivt/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'galasa.manager.ivt'
+}
+
+description = 'Galasa TSQ Manager IVTs'
+
+version = '0.1.0'
+
+dependencies {
+    implementation project (':galasa-managers-core-parent:dev.galasa.core.manager')
+    implementation project (':galasa-managers-cicsts-parent:dev.galasa.cicsts.tsq.manager')
+    implementation project (':galasa-managers-zos-parent:dev.galasa.zos3270.manager')
+}
+
+
+// Note: These values are consumed by the parent build process
+// They indicate which packages of functionality this OSGi bundle should be delivered inside,
+// or referenced from.
+// The settings here are gathered together by the build process to create a release.yaml file 
+// which gathers-up all the packaging metadata about all the OSGi bundles in this component.
+ext.projectName=project.name
+ext.includeInOBR          = true
+ext.includeInMVP          = true
+ext.includeInBOM          = true
+ext.includeInIsolated     = true
+ext.includeInCodeCoverage = false
+ext.includeInJavadoc      = false
```

Comment:
Should be true for javadoc if these APIs are to appear on the javadoc site.

Reply:
Hi @techcobweb , Is this required for IVTs?